README.md

```diff
@@ -83,37 +83,262 @@ pip install sparsezoo
+
+- `checkpoint_prepruning`
+- `checkpoint_postpruning`
+- `checkpoint_preqat`
+- `checkpoint_postqat` 
+
+The checkpoint that the model defaults to is the `preqat` state (just before the quantization step).
+
+```python
+from sparsezoo import Model
+
+stub = "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant_3layers-aggressive_84"
+
 model = Model(stub)
-print(model)
+available_checkpoints = model.training.available
+print(available_checkpoints)
+
+>> ['preqat']
+
+preqat_checkpoint = model.training.default # recipe defaults to `preqat`
+preqat_checkpoint_path = preqat_checkpoint.path # downloads the checkpoint and returns its path
+print(preqat_checkpoint_path)
+
+>> .../.cache/sparsezoo/0857c6f2-13c1-43c9-8db8-8f89a548dccd/training
+
+[print(file.name) for file in preqat_checkpoint.files]
+
+>> vocab.txt
+>> special_tokens_map.json
+>> pytorch_model.bin
+>> config.json
+>> training_args.bin
+>> tokenizer_config.json
+>> trainer_state.json
+>> tokenizer.json
 ```
 
-#### Searching the Zoo
+
+#### Accessing Recipes (Through Stub String Arguments)
+
+You can also directly request a specific recipe/checkpoint type by appending the appropriate URL query arguments to the stub:
+```python
+from sparsezoo import Model
+
+stub = "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe=transfer"
+
+model = Model(stub)
+
+# Inspect which files are present.
+# Note that the available recipes are restricted
+# according to the specified URL query arguments
+print(model.recipes.available)
+
+>> ['transfer-classification']
+
+transfer_recipe = model.recipes.default # Now the recipes default to the one selected by the stub string arguments
+print(transfer_recipe)
+
+>> File(name=recipe_transfer-classification.md)
+```
+
+### Accessing Sample Data
+
+The user may easily request a sample batch of data that represents the inputs and outputs of the model.
+
+```python
+sample_data = model.sample_batch(batch_size = 10)
+
+print(sample_data['sample_inputs'][0].shape)
+>> (10, 3, 224, 224) # (batch_size, num_channels, image_dim, image_dim)
+
+print(sample_data['sample_outputs'][0].shape)
+>> (10, 1000) # (batch_size, num_classes)
+```
+
+### Model Search
+The function `search_models` enables the user to quickly filter the contents of SparseZoo repository to find the stubs of interest:
 
 ```python
 from sparsezoo import search_models
 
-models = search_models(
-    domain="cv",
-    sub_domain="classification",
-    return_stubs=True,
-)
-print(models)
+args = {
+    "domain": "cv",
+    "sub_domain": "segmentation",
+    "architecture": "yolact",
+}
+
+models = search_models(**args)
+[print(model) for model in models]
+
+>> Model(stub=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none)
+>> Model(stub=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned90-none)
+>> Model(stub=zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none)
 ```
 
+
```

Comment:
Is the end of the next paragraph missing some words or was this phrase left in? e.g., 
Default `~/.cache/sparsezoo/`

Reply:
Nope, it is just an empty line.